lab2/ssh_deploy-apps.sh

```diff
@@ -0,0 +1,33 @@
+SERVER_HOST_DIR=$(pwd)/nestjs-rest-api
+CLIENT_HOST_DIR=$(pwd)/shop-angular-cloudfront
+
+# destination folder names can be changed
+SERVER_REMOTE_DIR=/var/app/lab2_server
+CLIENT_REMOTE_DIR=/var/www/lab2_client
```

Comment:
as you can see here 'var/www' folder is used to deploy client app.

So this line can be changed to the same 'www' floder
https://github.com/DzianisPasiukou/devops-fundamentals-course/pull/4/files#diff-ef283b770a1d16f6359421ac9bad4f3f5baa2d5e54782ef2f0c7527846a29037R34